contracts/hardhat.config.ts

```diff
@@ -80,19 +84,39 @@ const config: HardhatUserConfig = {
     tests: './test',
   },
   solidity: {
-    version: '0.6.12',
-    settings: {
-      // https://hardhat.org/hardhat-network/#solidity-optimizer-support
-      optimizer: {
-        enabled: true,
-        runs: 200,
+    compilers: [
+      {
+        version: '0.7.6',
+        settings: {
+          // https://hardhat.org/hardhat-network/#solidity-optimizer-support
+          optimizer: {
+            enabled: true,
+            runs: 10000,
+          },
+        },
       },
-    },
+      {
+        version: '0.6.12',
+        settings: {
+          // https://hardhat.org/hardhat-network/#solidity-optimizer-support
+          optimizer: {
+            enabled: true,
+            runs: 10000,
+          },
+        },
+      },
+    ],
```

Comment:
Small gas savings if we bump the runs to 999,999. If we bump to 9,999,999 it doesn't really make a difference, so I'd suggest we use 999,999 runs for both solc versions